spec/abstractops.html

```diff
@@ -294,6 +294,49 @@ <h1>RoundNumberToIncrement ( _x_, _increment_, _roundingMode_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-roundtime" aoid="RoundTime">
+    <h1>RoundTime ( _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_, _increment_, _unit_, _roundingMode_ )</h1>
+    <emu-alg>
+      1. Let _fractionalSecond_ be _nanosecond_ × 10<sup>−9</sup> + _microsecond_ × 10<sup>−6</sup> + _millisecond_ × 10<sup>−3</sup> + _second_.
```

Comment:
Are these Numbers or mathematical values?

Reply:
Good catch, they need to go from Numbers to mathematical values here. I'd like to avoid a bunch of extra variables or repeated lines so I went with `Let _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_, and _increment_ each be the mathematical values of themselves.` but I'm not 100% sure if it's rigorous enough.